src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static core.basesyntax.FileCreator.createFile;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
-    // HINT: In the `public static void main(String[] args)` it is better to create instances of your classes, 
-    // and call their methods, but do not write any business logic in the `main` method!
+    public static void main(String[] args) throws IOException {
```

Comment:
The `main()` method should be refactored to demonstrate the independence of services. Currently, it seems like the 'createFile()' method is being called without being part of any service. It should be part of a service and called accordingly.